sindresorhus/refined-github
# Restore some features that appear in files and file lists

Fixes #3945 

Missing:

- [x] `forked-to`
- [x] `fork-source-link-same-view`
- [ ] any more features affected? Probably every isRepoTree/isSingleFile


## Test 1

![gif](https://user-images.githubusercontent.com/1402241/114991374-b5276000-9ec3-11eb-9a8d-72129d3da0f2.gif)


1. Visit https://github.com/sindresorhus/eslint-plugin-unicorn/tree/main/docs
2. Notice `edit-files-faster` and `latest-tag-button`
3. Enter the folder
4. The same features are still working
5. Open a markdown file
6. Notice `view-markdown-source` (wrong position due to https://github.com/sindresorhus/refined-github/pull/4243)

## Test 2

![gif](https://user-images.githubusercontent.com/1402241/114993778-57e0de00-9ec6-11eb-8e9d-5a908cd04b69.gif)


1. Visit https://github.com/sindresorhus/refined-github/tree/main/source/features
2. Open a tsx file
3. Notice `copy-file` and `rgh-feature-description`

## Test 3

![gif](https://user-images.githubusercontent.com/1402241/114994696-3c2a0780-9ec7-11eb-8159-97ddc3a6af61.gif)

1. Visit https://github.com/sindresorhus/refined-github/tree/main/source
2. Enter a folder
3. Notice `download-folder-button` and `default-branch-button`
